programs/sbf/Cargo.toml

```diff
@@ -200,3 +200,15 @@ targets = ["x86_64-unknown-linux-gnu"]
 # overrides in sync.
 solana-program = { path = "../../sdk/program" }
 solana-zk-token-sdk = { path = "../../zk-token-sdk" }
+
+#
+# === zeroize versioning issues ===
+#
+# See `../../Cargo.toml` for details.  The following patches allow us to build
+# the whole repository with a newer version of `zeroize` than 1.3.
+[patch.crates-io.aes-gcm-siv]
+git = "https://github.com/RustCrypto/AEADs"
+rev = "6105d7a5591aefa646a95d12b5e8d3f55a9214ef"
```

Comment:
Looks like there is a newer commit that removes the upper bound in the issue raised and discussed [here](https://github.com/RustCrypto/utils/issues/723). The updated commit [here]( https://github.com/RustCrypto/nacl-compat/commit/992e4ca6015ad10daf7fbd64bd730e86f6101291). I know the compatibility issue was around needing >1.4 and this commit removes the <1.5. But may be worth keeping if versioning requirements change in the future.